src/compose/utils.ts

```diff
@@ -344,13 +344,19 @@ export async function addFeaturesFromLabels(
 				? service.config.volumes.push('/lib/firmware:/lib/firmware')
 				: null,
 		'io.balena.features.balena-socket': () => {
+			// Mount in /host/run to avoid clashing with systemd
+			const dstDockerSocket = '/host/run/balena-engine.sock';
```

Comment:
We can probably stick this in `./src/lib/constants.ts` I think

Reply:
I thought about it, but `constants.ts` seems to be more related to values related to the host. I'll move it